### ISSUE TITLE
ng generate component ignores angular.json settings after @cypress/schematic is installed

### ISSUE THREAD

**Issue by JasonWeinzierl** [cypress-io/cypress#25875]
### Current behavior

When generating a new component with `ng generate component`, the Angular CLI ignores schematic settings in `angular.json`.

For example, if I have

```json
"schematics": {
  "@schematics/angular:component": {
    "style": "scss",
    "changeDetection": "OnPush"
  }
}
```

and generate a component after @cypress/schematic is installed, the new component will not have SCSS styles nor OnPush change detection.

Output before installing @cypress/scematic:

```
CREATE src/app/my-component/my-component.component.html (27 bytes)
CREATE src/app/my-component/my-component.component.spec.ts (635 bytes)
CREATE src/app/my-component/my-component.component.ts (226 bytes)
CREATE src/app/my-component/my-component.component.scss (0 bytes)
UPDATE src/app/app.module.ts (497 bytes)  
```

Output after installing:

```
CREATE src/app/my-other-component/my-other-component.component.html (33 bytes)
CREATE src/app/my-other-component/my-other-component.component.ts (248 bytes)
CREATE src/app/my-other-component/my-other-component.component.css (0 bytes)
UPDATE src/app/app.module.ts (623 bytes)                                        
```

I was able to manually fix this by re-ordering the items in `schematicCollections` at the bottom of `angular.json`.

Instead of this:

```json
"cli": {
  "schematicCollections": [
      "@cypress/schematic",
      "@angular-eslint/schematics",
      "@schematics/angular"
  ]
}
```

I changed it to this:

```json
"cli": {
  "schematicCollections": [
      "@schematics/angular",
      "@cypress/schematic",
      "@angular-eslint/schematics"
  ]
}
```

### Desired behavior

Cypress angular schematics should not prevent `ng generate` from honoring settings in angular.json.

### Test code to reproduce

`ng new testproject --style scss --routing true`

`cd testproject`

`ng g c MyComponent`

A new component will have been generated with SCSS styling.

`ng add @cypress/schematic --skip-confirmation --e2e --component`

`ng g c MyOtherComponent`

A new component will be generated with CSS styling instead of SCSS.

### Cypress Version

12.6.0

### Node version

18.4.1

### Operating System

Windows 10 22H2 19045.2604

**Comment by ray1618** [comment]
We had the same issue and fixed it by adding style: scss to @cypress/schematic:component in the schematics

```"
schematics": {
        "@schematics/angular:component": {
          "style": "scss"
        },
        "@cypress/schematic:component": {
          "style": "scss"
        }
      },
```

**Comment by marktnoonan** [comment]
Thanks @JasonWeinzierl for posting and @ray1618 for offering a workaround. I haven't had a chance to test this but hope to in the next couple of days, certainly seems like something we'd want to correct on our side.

**Comment by jordanpowell88** [comment]
Yeah this should most certainly be resolved. Thank you @JasonWeinzierl for opening the issue

**Comment by marktnoonan** [comment]
Going to go ahead and route this one to the CT team

**Comment by ana-ac** [comment]
I've created an issue with what's happening to me in the angular-cli repository [#30165](https://github.com/angular/angular-cli/issues/30165), which is the same as described by [@JasonWeinzierl](https://github.com/JasonWeinzierl) here https://github.com/cypress-io/cypress/issues/25875#issue-1590441513. It still doesn't seem to be fixed.
